<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>524</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Pendulum Simulator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="lengthSpinBox">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QDoubleSpinBox" name="intervalSpinBox">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="value">
               <double>0.012500000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="gSpinBox">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>-9.810000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Pivot Coordinates (x,y):</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Initial Angular Velocity:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Type in paramters, they're in SI units:</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="pivotCoorLineEdit">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>0,0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Gravitational Accleration:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="angleSpinBox">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>-0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Time Interval:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Pendulum Length:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Initial Angle (in terms of pi):</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="angularVelocitySpinBox">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="applyButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Simulate</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="changesLabel">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;
color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>To save, type in full file path, including .mp4:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="saveLabel">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Arial&quot;;
color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="vLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
